Manuale tecnico con sezioni aggiornate

GMindMercatis opened the pull request:
Manuale tecnico con overview, saml, oidc e sdk separati

GMindMercatis committed:
Update Sezioni Manuale Tecnico

GMindMercatis committed:
immagini aggiornate

GMindMercatis committed:
link ai doc

GMindMercatis committed:
Aggiunto conf.py

GMindMercatis committed:
Bug Fixing PDF

aantetomaso committed:
Revisione per pubblicazione

Ho inserito alcune revisioni di forma.

fmarino-ipzs committed:
fix-ipzs: solved ecoded char error + minor typos

fmarino-ipzs committed:
fix-ipzs: removed html output folder

fmarino-ipzs committed:
fix-ipzs: update gitignore

fmarino-ipzs committed:
chore-ipzs: editorial

fmarino-ipzs commented on overview.rst:
```diff
@@ -1,129 +1,316 @@
+logico della soluzione eID basata sulla CIE.
 
-   Schema di autenticazione Entra con CIE.
+.. figure:: media/image1.png
```
Modificare questa immagine, fa riferimento all'autenticazione con liv3. O si rende più generica oppure si inseriscono riferimenti anche a liv1 e 2. @aantetomaso @GMindMercatis che ne pensate?

aantetomaso replied:
Penso sia una immagine vecchia, nella nuova versione non c'è nè il file nè la referenza

fmarino-ipzs replied:
di quale versione parli? Mi pare che sia una immagine presente nel testo e disponibile qui: media/image1.png

fmarino-ipzs commented on overview.rst:
```diff
@@ -1,129 +1,316 @@
+   “\ *basso*\ ” (livello 1);
 
-   Schema di autenticazione Entra con CIE tramite modalitá "desktop".
+2. secondo fattore di autenticazione: costituito da un codice OTP
```
Sarebbe possibile spiegare meglio questo punto? @GMindMercatis

GMindMercatis replied:
aggiornato

fmarino-ipzs commented on overview.rst:
```diff
@@ -1,129 +1,316 @@
+   o `iOS <https://apps.apple.com/it/app/cieid/id1504644677>`__).
 
-   Schema di autenticazione Entra con CIE tramite modalitá "mobile".
+Allo stato dell'arte questa modalità è fruibile mediante smartphone
```
@aantetomaso, possiamo verificare che non ci siano aggiornamenti su supporto OS/Browser?

aantetomaso replied:
Le versioni sono corrette, tuttavia L2 è fruibile anche senza NFC. Preciso meglio.

aantetomaso replied:
Precisato meglio nel file, appena possibile effettuo il commit.

fmarino-ipzs replied:
Perfetto grazie mille

fmarino-ipzs commented on overview.rst:
```diff
@@ -1,129 +1,316 @@
+**DESKTOP**
 
-   Pulsante ufficiale "Entra con CIE"
+.. figure:: media/image3.png
```
@GMindMercatis @aantetomaso , mi pare di vedere che i due sequence diagram relativi al liv desktop o smartphone sono identici (e direi che ha senso dal mio punto di vista). Quindi suggerirei di eliminare uno dei due e dire che si riferiscono appunto a entrambi gli scenari. Che ne dite?

fmarino-ipzs replied:
Inoltre suggerirei l'utilizzo di https://sequencediagram.org/ così possiamo inserire poi il link alla preview e facilitare la lettura del diagramma (oltre al fatto che ha una qualità molto alta)

fmarino-ipzs replied:
Altra cosa: visto che alla fine quello che effettivamente cambia nei vari scenari è la parte di challenge con utente, potremmo pensare di riportare uno schema generale all'inizio con una box che poi andiamo ad espandere nei vari sotto casi.

aantetomaso replied:
Concordo nell'eliminazione del secondo diagramma (ho proceduto a modificare), meno sull'utilizzo dell'altro tool, come formalismo e resa preferisco Visio. 
Ad ogni modo, come preferite.

GMindMercatis replied:
Antonio non vedo ancora tue modifiche su questo file...non hai ancora fatto push?

aantetomaso replied:
Non ancora, ho difficoltà in azienda. Cerco di farlo stasera.

aantetomaso replied:
Sono riuscito ora.

fmarino-ipzs committed:
chore-ipzs: editorial

fmarino-ipzs commented on LICENSE:
Credo che vada mantenuto ma in caso lo rimettiamo dopo che abbiamo unito

fmarino-ipzs commented on document_settings.yml:
@GMindMercatis ripristinare questo file con il seguente contenuto:
--- 
document: 
  name: "Manuale Tecnico per gli erogatori di servizi pubblici e privati"
  short_name: cie-manuale-tecnico
  description: "Manuale tecnico rivolto agli erogatori di servizio che fornisce una overview ad alto livello sulle modalità di funzionamento e integrazione dello schema di identificazione Entra con CIE per l'accesso ai servizi in rete di PP.AA e Privati."
  tags: 
    - Manuale tecnico
    - Entra con CIE
    - CIE
    - Carta di Identità Elettronica
    - Docs Italia

GMindMercatis replied:
fatto

fmarino-ipzs commented on media/image3.png:
questa immagine è identica alla image4

aantetomaso replied:
Ho trovato il file image1, guardavo il repository sbagliato. In realtà quell'immagine raffigura tutto lo schema e l'insieme di componenti software (carta, PC, lettore NFC, app), a prescindere dal livello di sicurezza. Dato il livello alto che tiene, lo manterrei così, perchè tanto i diagrammi successivi spiegano come si svolge il flusso.

fmarino-ipzs reviewed (changes requested):
Ho effettuato alcune modifiche e inserito dei commenti. Resto in attesa di un vostro feedback sui commenti aperti.

aantetomaso committed:
Recepimento revisioni di Francesco

1) Eliminato diagramma duplicato per L1 in caso di desktop e smartphone
2) Precisato meglio passaggio inerente a browser e versioni SO

GMindMercatis committed:
Update  autenticazione a due fattori overview.rst
